FirebaseStorageSwift/Sources/SwiftApiExtension.swift

```diff
@@ -27,8 +27,9 @@ private func getResultCallback<T>(
     } else {
       completion(.failure(NSError(domain: "FirebaseStorageSwift",
                                   code: -1,
-                                  userInfo: ["Storage Result Generator":
-                                    "InternalError - Return type and Error code both nil"])))
+                                  userInfo: ["NSLocalizedDescription":
```

Comment:
nit: Instead of using a string literal, use [`NSLocalizedDescriptionKey`](https://developer.apple.com/documentation/foundation/nslocalizeddescriptionkey).